guardian/master-to-main
# ci: Enable Combine PRs workflow with custom GH token

## What does this change?
Following the changes in #258, work through [the instructions](https://github.com/github/combine-prs/blob/main/docs/github-app-setup.md) to configure the Workflow to allow the resulting PR to trigger a build.

I've created https://github.com/apps/gu-combine-prs for this. It is currently only installed on this repository. We should widen the scope if we find value in this automation, and wish to roll it out across the organisation.

## How to test
- Trigger the workflow from `main`
- Observe the [resulting PR](https://github.com/guardian/master-to-main/pull/259) not triggering a build
- Trigger the workflow from this branch
- Observe the [resulting PR](https://github.com/guardian/master-to-main/pull/261) triggering a build

## How can we measure success?
Running a build increases confidence that the grouped changes from the combined PRs work together as we can observe the build status. If the build passes, the combined PR is good to go.
